src/main/java/org/sagebionetworks/web/server/servlet/filter/CrawlFilter.java

```diff
@@ -660,4 +659,19 @@ public EntityChildrenRequest createGetChildrenQuery(String parentId) {
     newQuery.setIncludeTypes(types);
     return newQuery;
   }
+
+  /**
+   * Helper to convert from the non-gwt compatible PaginatedResults to the compatible type.
+   *
+   * @param in
+   * @return
+   */
+  public <T extends JSONEntity> PaginatedResults<T> convertPaginated(
```

Comment:
Brought this over from DiscussionForumClient to eliminate dependency but reduce the changes to this class